lesson_19/zoldi_razvan/lesson_19_zoldi_razvan.py

```diff
@@ -0,0 +1,73 @@
+import contextlib
+
+#  *******************************  1st approach   **********************************
+# considering that we should handle KeyError I create a dictionary and because we should also
+#  handle IndexError I took into consideration a LIST of dictionaries
+
+# @contextlib.contextmanager
```

Comment:
is the commented code working?

Reply:
well , it should. is not?

Reply:
I've changed a little bit first part and put 2 with's to see error handling